src/android/toga_android/widgets/textinput.py

```diff
@@ -66,5 +66,7 @@ def rehint(self):
         self.native.measure(
             View__MeasureSpec.UNSPECIFIED, View__MeasureSpec.UNSPECIFIED
         )
-        self.interface.intrinsic.width = at_least(self.native.getMeasuredWidth())
+        # Use arbitrary minimum 100px width for TextInput.
+        self.interface.intrinsic.width = at_least(100)
```

Comment:
On other platforms, we've defined a MIN_WIDTH as a constant on the base widget so that this min size isn't a *completely* arbitrary constant (or, at least, it's a consistently defined one)

Reply:
Makes sense to me! I'll do it.

Reply:
Done -- see latest push.